packages/dds/tree/src/feature-libraries/editable-tree/editableTreeContext.ts

```diff
@@ -308,13 +305,16 @@ export class ProxyContext implements EditableTreeContext {
  * A simple API for a Forest to interact with the tree.
  *
  * @param forest - the Forest
- * @param transactionCheckout - the TransactionCheckout applied to a transaction, not required in read-only usecases.
+ * @param runTransaction - a function which executes a given transaction, returning true if it was successful or false if aborted. Not required in read-only use-cases.
  * @returns {@link EditableTreeContext} which is used to manage the cursors and anchors within the EditableTrees:
  * This is necessary for supporting using this tree across edits to the forest, and not leaking memory.
  */
 export function getEditableTreeContext(
 	forest: IEditableForest,
-	transactionCheckout?: TransactionCheckout<DefaultEditBuilder, DefaultChangeset>,
+	runTransaction?: (
+		/** A function which performs a transaction, returning true if the transaction should succeed or false if it should be aborted */
```

Comment:
this isn't a good way to document this. Its redundant with the param doc above and the other copy of this comment further above.

Either rely on the param doc, and/or define a type for it and but the docs on that type.

Reply:
Note that if you pass in just the editor as suggested above, this goes away